main/main.cpp

```diff
@@ -599,6 +599,7 @@ void Main::print_help(const char *p_binary) {
 	print_help_option("--rendering-method <renderer>", "Renderer name. Requires driver support.\n");
 	print_help_option("--rendering-driver <driver>", "Rendering driver (depends on display driver).\n");
 	print_help_option("--gpu-index <device_index>", "Use a specific GPU (run with --verbose to get a list of available devices).\n");
+	print_help_option("--renderer-latency <mode>", "Override the renderer low latency mode [\"high_framerate\", \"low_latency\"].\n");
```

Comment:
I wonder if we should rename the argument to `--rendering-latency` to match the existing `--rendering-method` and `--rendering-driver` arguments. This should reduce the risk of typos when entering the CLI argument name, which can be problematic as Godot doesn't warn when using unknown command line arguments.

```suggestion
	print_help_option("--rendering-latency <mode>", "Override the renderer low latency mode [\"high_framerate\", \"low_latency\"].\n");
```